### DISCUSSION TITLE
limb health system

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#8654] [Ideas and suggestions]
I suggest implementing a limb health system, to me it doesn't make sense that you can either randomly split an endworm by the first limb with a default shell and kill it in seconds, or shoot it in the last limb with physicorium shells 20 times and not put a dent on it.

Adding this system would decrease the rng factor of any creature that uses it, making combat more consistent and skill rewarding.

Maybe on certain enemies, only a specific type of ammo/weapon could damage the limb itself (example: endworm's limbs only damaged by railgun, the endworm is split when the limb's health reaches 0).

**Comment by itchyOwl** [comment]
I completely agree, but it's a matter of time and resources and where to put them. If we'd going to touch it, a health based system would be better than the current. But I already knew that when the current system was (relatively quickly) implemented.

**Comment by Ek-Videogames** [comment]
Unfortunately couldn't find one of my posts from years ago, in short: Yes, limb severance based on accumulated damage - this would also allow weapons with low single-shot damage to still be able to break limbs under constant firing DPS.

The idea of making enemies invulnerable to certain weapons entirely doesn't sit well, unless you mean like the current system where certain limbs will still have a minimum required damage to sever. Whatever decision is made, it should be taken into account that not every vanilla sub even has railguns.
